website/content/v1.7/talos-guides/install/boot-assets.md

```diff
@@ -91,6 +95,64 @@ Same way upgrade process can be used to transition to a new set of system extens
 talosctl upgrade --image factory.talos.dev/installer/<new_schematic_id>:{{< release >}}
 ```
 
+### Example: Raspberry Pi generic with Image Factory
+
+Let's assume we want to boot Talos on a Raspberry Pi with `iscsi-tools` system extension.
+
+First, let's create the schematic file `rpi_generic.yaml`:
+
+```yaml
+# rpi_generic.yaml
+overlay:
+  name: rpi_generic
+  image: siderolabs/sbc-raspberrypi
+customization:
+  systemExtensions:
+    officialExtensions:
+      - siderolabs/iscsi-tools
+```
+
+> The schematic doesn't contain any system extension or overlay versions, Image Factory will pick the correct version matching Talos Linux release.
+
+And now we can upload the schematic to the Image Factory to retrieve its ID:
+
+```shell
+$ curl -X POST --data-binary @rpi_generic.yaml https://factory.talos.dev/schematics
+{"id":"0db665edfda21c70194e7ca660955425d16cec2aa58ff031e2abf72b7c328585"}
+```
+
+The returned schematic ID `0db665edfda21c70194e7ca660955425d16cec2aa58ff031e2abf72b7c328585` we will use to generate the boot assets.
+
+> The schematic ID is based on the schematic contents, so uploading the same schematic will return the same ID.
+
+Now we can download the metal arm64 image:
+
+* https://factory.talos.dev/image/0db665edfda21c70194e7ca660955425d16cec2aa58ff031e2abf72b7c328585/{{< release >}}/metal-arm64.raw.xz (download it and burn to a boot media)
+
+> The Image Factory URL contains both schematic ID and Talos version, and both can be changed to generate different boot assets.
+
+Once the Raspberry Pi is booted up for the first time, it will require Talos Linux `installer` image to be installed on the disk.
```

Comment:
as it's a disk image, it doesn't need an installer actually

Reply:
ah yes, i'll add a section on upgrading if it's not already there